pkgs/by-name/bl/blackbeard/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  buildGoModule,
+  lib,
+  fetchFromGitHub,
+  curl-impersonate-chrome,
+  pkg-config,
+  curl,
+}:
+
+buildGoModule rec {
+  pname = "blackbeard";
+  version = "0-unstable-2024-01-02";
+
+  src = fetchFromGitHub {
+    owner = "matheusfillipe";
+    repo = "BlackBeard";
+    rev = "788be485df436bf009e716c9918c395bda8d86f4";
+    hash = "sha256-2i7r3asDBEBSUVOR+n3C+DH2HLeE2sbVJp95jfE/UpQ=";
+  };
+
+  vendorHash = "sha256-4cUoI8OcBCUT3uqvmIduCj0My8LCVUoDvvAS21DR2Mo=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  preBuild = ''
+    export TEMP=$(mktemp -d)
+    pushd $TEMP
+    cp ${curl.dev}/lib/pkgconfig/libcurl.pc .
+    substituteInPlace libcurl.pc \
+      --replace-fail "${curl.out}" "${curl-impersonate-chrome}" \
+      --replace-fail "-lcurl" "-lcurl-impersonate-chrome"
```

Comment:
This is perhaps a bit silly, but we should probably explain what is happening here. (curl-impersonate-chrome being a fork of curl emulating real browsers, and having to patch curl's existing pkgconfig file since curl-impersonate-chrome doesn't come with one)